ports/stm/supervisor/usb.c

```diff
@@ -79,7 +78,11 @@ void init_usb_hardware(void) {
     GPIO_InitStruct.Speed = GPIO_SPEED_HIGH;
     GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
     GPIO_InitStruct.Pull = GPIO_NOPULL;
+    #if defined(STM32H7)
+    GPIO_InitStruct.Alternate = GPIO_AF10_OTG1_FS;
+    #else
```

Comment:
Are the `#else`s here and below for F4? I'd `#if`-check for the F4 family explicitly rather than use `#else`, since if yet another family is added, these might be wrong.